docs/site/Parsing-requests.md

```diff
@@ -234,6 +234,76 @@ class Product extends Entity {
 A full list of options & usage scenarios could be found in the
 [documentation of AJV errors](https://github.com/epoberezkin/ajv-errors).
 
+##### Custom AJV Formats
+
+AJV Custom Formats can be configured by binding the value
+`validation.ajvCustomFormats` to `RestBindings.REQUEST_BODY_PARSER_OPTIONS`
+('rest.requestBodyParserOptions').
+
+For example,
+
+```ts
+server.bind(RestBindings.REQUEST_BODY_PARSER_OPTIONS).to({
+  validation: {
+    ajvCustomFormats: {
```

Comment:
I think it should be named something else other than `ajvCustomFormats` to provide abstraction on the `AJV` module (The `pattern` key in `@properties` does not have the `ajv` prefix)

Reply:
The key is meant to provide details for creating a new custom format for ajv. As of now, they are meant to be for ajv thus name `ajvCustomFormats`.

Reply:
@mastermunj I understand that they're meant for AJV, I just think we need to create an abstraction layer from AJV so that we have more control over the APIs exposed to the developers. Postponing the change would mean a breaking change (major semver).

Reply:
I understand your point, however, ajv is in a way tightly integrated into LB4 schema validation as of now. `ajvKeywords` & `ajvErrors` are examples of it.

I believe, we will need to look at the inclusion of ajv as a whole for achieving the abstraction layer.

Reply:
I agree that AJV is already tighly-integrated as per-your examples; However, the APIs exposed aren't prefixed with `ajv`. (e.g. `ajvErrors` is exposed as `jsonSchema.errorMessage`). I just think it should be named something else.

Reply:
> I think it should be named something else other than `ajvCustomFormats`

I share the same opinion. The interface of AJV should be abstracted as much as possible. The key should have a generic name, independent of the underlying library; there is no guarantee we will never switch the validation library.